modules/appeals_api/app/models/appeals_api/supplemental_claim.rb

```diff
@@ -31,6 +31,114 @@ def self.date_from_string(string)
       if: proc { |a| a.form_data.present? }
     )
 
+    def veteran_first_name
+      auth_headers.dig('X-VA-First-Name')
+    end
+
+    def veteran_middle_initial
+      auth_headers.dig('X-VA-Middle-Initial')
+    end
+
+    def veteran_last_name
+      auth_headers.dig('X-VA-Last-Name')
+    end
+
+    def full_name
+      "#{veteran_first_name} #{veteran_middle_initial} #{veteran_last_name}".squeeze(' ').strip
+    end
+
+    def ssn
+      auth_headers.dig('X-VA-SSN')
+    end
+
+    def file_number
+      auth_headers.dig('X-VA-File-Number')
+    end
+
+    def veteran_dob_month
+      birth_date.strftime '%m'
+    end
+
+    def veteran_dob_day
+      birth_date.strftime '%d'
+    end
+
+    def veteran_dob_year
+      birth_date.strftime '%Y'
+    end
+
+    def veteran_service_number
+      auth_headers.dig('X-VA-Service-Number')
+    end
+
+    def insurance_policy_number
+      auth_headers.dig('X-VA-Insurance-Policy-Number')
+    end
+
+    def mailing_address_number_and_street
+      veteran.dig('address', 'addressLine1') || ''
+    end
+
+    def mailing_address_apartment_or_unit_number
+      veteran.dig('address', 'addressLine2') || ''
+    end
+
+    def mailing_address_box
```

Comment:
I don't think we should assume what these address lines represent. It's not guaranteed that every PO Box will come in on `addressLine3` - if the address is only their PO Box, they'd likely put it in `addressLine1`. I think it's fine to keep these method names tied to their schema fields.